packages/appstore/src/components/modals/real-wallets-upgrade/components/ready-to-upgrade-wallets/ready-to-upgrade-wallets.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Checkbox, Text, Icon } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import WalletsImage from 'Assets/svgs/wallets';
+import getUpgradeInformationList from 'Constants/upgrade-info-lists-config';
+import './ready-to-upgrade-wallets.scss';
+import { useContentFlag } from '@deriv/hooks';
+
+type TReadyToUpgradeWallets = {
+    value: boolean;
+    toggleCheckbox: () => void;
+};
+
+const ReadyToUpgradeWallets = observer(({ value, toggleCheckbox }: TReadyToUpgradeWallets) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    const text_body_size = is_mobile ? 'xs' : 's';
+    const text_info_size = is_mobile ? 'xxs' : 'xs';
+    const form_line_height = is_mobile ? 'm' : 'l';
+
+    const { is_eu_demo, is_eu_real, is_low_risk_cr_eu } = useContentFlag();
+    const is_eu = is_eu_demo || is_eu_real || is_low_risk_cr_eu;
```

Comment:
just curious, whats the difference between using `useContentFlag` and the `is_eu` computed value from client-store, should we not use the client store value as we need to only check if the user is from eu right?

Reply:
The discussion on the slack thread suggests is_eu and is_eu_user from the store is the same thing but I am not sure what should be the preference (probably flag from the store) but still I am not sure if I change the implementation how it would impact the business logic here.

Reply:
i think its ok to use the hook in this case since we want to slowly phase out stores 👍